applications_tests/lethe-fluid/tracer_inlet_dirichlet_bc.prm

```diff
@@ -0,0 +1,121 @@
+set dimension = 2
```

Comment:
It would be better to use a test case for which you can have an analytical solution or something similar to verify that your implementation is good.